src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -202,11 +202,11 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNet6Win)'">
     <UseWindowsForms>true</UseWindowsForms>
     <!-- 
-    MSAL will run on Windows 7 and 8 but requires to be built against Win10 to use WinRT APIs for WAM 
+    MSAL will run on Windows 8 but requires to be built against Win10 to use WinRT APIs for WAM 
     See https://docs.microsoft.com/en-us/dotnet/standard/analyzers/platform-compat-analyzer and 
     https://github.com/dotnet/designs/blob/main/accepted/2020/platform-checks/platform-checks.md for details
     -->
-    <SupportedOSPlatformVersion>7</SupportedOSPlatformVersion>
+    <SupportedOSPlatformVersion>8</SupportedOSPlatformVersion>
```

Comment:
win 7 is deprecated and this is causing a warning.

Reply:
It's still giving an error:
> warning CA1418: Version '8' is not valid for platform 'Windows'. Use a version with 2-4 parts for this platform.